controllers/ipfs_cluster_service.go

```diff
@@ -0,0 +1,20 @@
+package controllers
+
+import (
+	"github.com/ipfs-cluster/ipfs-cluster/cmdutils"
+	ma "github.com/multiformats/go-multiaddr"
+)
+
+func GetDefaultServiceConfig() *cmdutils.ConfigHelper {
+	cfgHelper := cmdutils.NewConfigHelper("", "", "crdt", "badger")
```

Comment:
We use flatfs for the datastore so that should also be set here. Maybe this can be parameterized?

Reply:
@coryschwartz Does IPFS Cluster maintain its own blockstore or does this value need to be in sync with what we use for the IPFS nodes?

Reply:
I have copied this from [ipfs-cluster](https://github.com/ipfs-cluster/ipfs-cluster/blob/38b463d65836bba818d9a1018416109c5d5bfec0/cmd/ipfs-cluster-service/main.go#L312) project. Default value is [badger](https://github.com/ipfs-cluster/ipfs-cluster/blob/master/cmd/ipfs-cluster-service/main.go#L40).
From the code - [here](https://github.com/ipfs-cluster/ipfs-cluster/blob/master/cmd/ipfs-cluster-service/main.go#L307) I can see flatfs is not a valid value.

Reply:
@hsanjuan We would need to use `flatfs` here as well correct? `ipfs-cluster/cmdutils` doesn't seem to support flatfs as a valid value for the datastore, will we need to add support for it?